docs/concepts.rst

```diff
@@ -70,7 +70,9 @@ to several objects.
 Authentication
 ==============
 
-Kinto doesn't provide users management, like user sign-up, password modification, etc.
+Kinto comes with an accounts API that allows user sign-up, password modification and user deletion. It allows you to configure an administrator account that has read/write access to all other accounts.
+
+By default, Kinto accounts are authenticated using basic authentication.
```

Comment:
> I don't think we should put the built-in account plugin too visible until it becomes enabled by default.

It was my understanding that by the time this branch is merged with master, this plugin would become the default and all related documentation needs to be updated to highlight this.

If this is not the case, it'd be helpful to have some pointers to which docs need to be updated.

My initial list consisted of the following:

- docs/core/quickstart.rst
- docs/concepts.rst
- docs/faq.rst
- docs/configuration/settings.rst
- docs/tutorials/first-steps.rst
- docs/api/1.x/authentication.rst